src/Storage/Commands.Storage/help/Remove-AzureStorageShare.md

```diff
@@ -37,6 +38,13 @@ PS C:\>Remove-AzureStorageShare -Name "ContosoShare06"
 
 This command removes the file share named ContosoShare06.
 
+### Example 2: Remove a file share and all its snapshots
+```
+PS C:\>Remove-AzureStorageShare -Name "ContosoShare06" -IncludeSnapshot
```

Comment:
IncludeAllSnapshot

Reply:
Good catch! The help has rebuild after change the parameter name from "IncludeSnapshot" to "IncludeAllSnaspshot" (agreed by feature team and OneSDK team), but the sample is missed. 
I have checked all other help change, no missing now.